mitosheet/src/components/taskpanes/AITransformation/AITransformationResultSection.tsx

```diff
@@ -97,7 +106,11 @@ const AITransformationResultSection = (props: AITransformationResultSectionProps
                     </div>
                 )
             })}
-            {!(result.last_line_value !== undefined && result.last_line_value !== null) && result.created_dataframe_names.length === 0 && Object.entries(result.modified_dataframes_column_recons).length === 0 && result.deleted_dataframe_names.length === 0 && 
+            {!(result.last_line_value !== undefined && result.last_line_value !== null) 
```

Comment:
We can change `{!(result.last_line_value !== undefined && result.last_line_value !== null) `
to `{(result.last_line_value === undefined || result.last_line_value === null)`

- result.last_line_value = undefined => true
- result.last_line_value = null => true 
- result.last_line_value  = something else => false